### PR TITLE
feat(dedicated): account contacts lazy loading

### PR DESCRIPTION
**Pull request by frenautvh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-6352, DTRSD-28678
| License          | BSD 3-Clause

## Description

Lazy load account/contacts section

### CODE REVIEW

**Review by JeremyDec — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/account/account.bundle.js [hunk 1]
```diff
@@ -1,23 +1,4 @@
 import './account.routing';
-import './contacts/request/change/user-contacts-request-change.controller';
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/account/account.bundle.js:2]: 🧹 🧹

**Review by marie-j — APPROVED** [review]

**Comment by frenautvh** [comment]
i just force pushed a commit to add missing imports :
`'./contacts/request/change/user-contacts-request-change.controller'` looks like it's not used because the corresponding template is never imported BUT in fact the template is included with a dynamic ng-include ...
a grep is not helpfull in that case to detect unused files so we have to be careful :/

**Review by ganeshkumar1989 — DISMISSED** [review]

**Review by marie-j — DISMISSED** [review]

**Review by marie-j — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]
Well done! Changes looks good to me.

Could you please update body commit in order to provide ticket reference in it?

Thank you!

**Comment by antleblanc** [comment]
Rewriting history.

**Comment by antleblanc** [comment]
History has been rewrite successfully.

**Review by antleblanc — APPROVED** [review]

**Pull request merged**